lms/urls.py

```diff
@@ -820,10 +820,16 @@
 
 if settings.FEATURES.get('ENABLE_OAUTH2_PROVIDER'):
     urlpatterns += (
+        # These URLs dispatch to django-oauth-toolkit or django-oauth2-provider as appropriate.
+        url(r'^oauth2/', include('lms.djangoapps.oauth_dispatch.urls')),
+        # These URLs contain the django-oauth2-provider default behavior (for views that have not
```

Comment:
You mean `django-oauth2-provider`?  (It gets confusing).  There are some views that we will never override.  For instance, in `django-oauth2-provider`, `/oauth2/authorize/` performs an immediate redirect to `/oauth2/authorize/confirm/`, for reasons that I'm not clear on.  `django-oauth-toolkit` performs no such redirect, so there's no need to write a wrapper around `/oauth2/authorize/confirm/`.  We can just leave that URL route around as an implementation detail of `django-oauth2-provider`.  

There are, however, follow up tickets to fill out the dispatch views that we need implemented (Authorize being the main one).